canonical/websites-content-system
# Add database config

## Done

 - Added database models and migrations
 - Added database results to the get-tree endpoint

## QA

### QA steps

 - Clone and run the repo, following the readme
 - Make a request to `/get-tree/canonical.com` and check that `owner` and `status` are part of the response 

## Fixes

 - Fixes #
 - Fixes [WD-13261](https://warthogs.atlassian.net/browse/WD-13261?atlOrigin=eyJpIjoiMjA3MjhiYTZhZjU3NDhlM2I0N2VlYjE1N2Q2NDE4ODYiLCJwIjoiaiJ9)

[WD-13261]: https://warthogs.atlassian.net/browse/WD-13261?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
